services/web/server/src/simcore_service_webserver/resource_manager/redis.py

```diff
@@ -35,11 +31,17 @@ async def redis_client(app: web.Application):
     async def create_client(url) -> aioredis.Redis:
         # create redis client
         client: Optional[aioredis.Redis] = None
-        async for attempt in AsyncRetrying(**retry_upon_init_policy):
+        async for attempt in AsyncRetrying(
+            stop=stop_after_delay(1 * _MINUTE),
+            wait=wait_random(max=10),
+            before_sleep=before_sleep_log(log, logging.WARNING),
+            reraise=True,
+        ):
             with attempt:
                 client = await aioredis.create_redis_pool(url, encoding="utf-8")
                 if not client:
                     raise ValueError("Expected aioredis client instance, got {client}")
+        assert client  # no sec
```

Comment:
let's start systematically logging when the connection is established after repeats, i.e. after the retry ... 
```python
for attempt in Retrying(..., reraise=True):
    for attempt:
          # access to another service

log.info(
   "Connection to %s succeded [%s]", 
    service_name, 
    json.dumps(attempt.retry_state.retry_object.statistics) 
) 
 

```